repo/content/committed_read_manager.go

```diff
@@ -335,6 +340,8 @@ func (sm *SharedManager) release(ctx context.Context) error {
 		return nil
 	}
 
+	sm.closed = true
```

Comment:
There is no guarantee that this write becomes visible elsewhere.

Reply:
fixed